packages/@tailwindcss-vite/src/index.ts

```diff
@@ -21,10 +21,13 @@ export default function tailwindcss(): Plugin[] {
     if (!server) return
 
     let updates: Update[] = []
-    for (let id of cssModules) {
+    for (let id of cssModules.values()) {
       let cssModule = server.moduleGraph.getModuleById(id)
       if (!cssModule) {
         console.log('Could not find css module', id)
```

Comment:
Good catch by Tim. We should probably drop this log then as well.

Reply:
Good call. Done.